src/main/java/lotto/domain/model/WinningLotto.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain.model;
+
+import lotto.domain.Rank;
+
+public record WinningLotto(LottoNumber bonusNumber, Lotto winNumbers) {
```

Comment:
```suggestion
public record WinningLotto(Lotto winNumbers, LottoNumber bonusNumber) {
```
저 같으면 로또를 첫 번째 인자로 구현할 것 같음